clinica/pipelines/statistics_surface/statistics_surface_utils.py

```diff
@@ -1,13 +1,30 @@
 # coding: utf8
 
-__author__ = "Junhao Wen"
-__copyright__ = "Copyright 2016-2019 The Aramis Lab Team"
-__credits__ = ["Junhao Wen"]
-__license__ = "See LICENSE.txt file"
-__version__ = "0.1.0"
-__maintainer__ = "Junhao Wen"
-__email__ = "Junhao.Wen@inria.fr"
-__status__ = "Development"
+
```

Comment:
- `author` and `version` are in `info.json`
- `credits` is not appropriate in this situation: the author should not credit himself but the people who helped the author
- `license` is redundant (it is the same for all Python files)
- `maintainer` is obsolete and I prefered delete it now instead of changing it every x month
- I find `status` not appropriate. When a pipeline or set of functions is considered finished?
- I don't like the `copyright` line because it can make people reluctant to contribute to Clinica

In general, I would be in favor of deleting these lines in Clinica.